src/app/ngrx/actions/castle.action.creator.ts

```diff
@@ -145,13 +196,13 @@ export interface CastleEpisodePerformanceMetricsSuccessPayload {
 }
 
 export class CastleEpisodePerformanceMetricsSuccessAction implements Action {
-  readonly type = ActionTypes.CASTLE_EPISODE_PERFORMANCE_METRICS_SUCCESS;
+  readonly type = <string>ActionTypes.CASTLE_EPISODE_PERFORMANCE_METRICS_SUCCESS;
```

Comment:
I got some TypeScript complaining when I moved the action types to string enums that casting to `<string>` in some places seemed to fix. 🤷‍♀️